python/py-cucumber-tag-expressions/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        cucumber tag-expressions-python ace002b
+version             2.0.2
+revision            0
+name                py-cucumber-tag-expressions
+categories-append   devel
+
+license             MIT
+maintainers         nomaintainer
+platforms           darwin
+
```

Comment:
please add ```supported_arch noarch```

Reply:
Thanks done. 🦃🦃🦃

Reply:
sorry, I see it’s my typo... it would still be caught locally by port lint

Reply:
Thanks done.